custom_components/tuya_local/devices/alecoair_d12_dehumidifier.yaml

```diff
@@ -46,19 +46,19 @@ primary_entity:
     - id: 19
       name: error
       type: bitfield
-      # Docs mention E1 and E2, it isn't clear which is tank full, so for now
-      # treat both as that.
+      # E2 might be a compressor fault.
       mapping:
         - dps_val: 0
           value: OK
+          icon: "mdi:cup-outline"
```

Comment:
Don't set the icon for the OK case. If the status is OK, you want the preset icons showing through.